build/build-lightrider.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const browserify = require('browserify');
+const cpy = require('cpy');
+const fs = require('fs');
+const path = require('path');
+const makeDir = require('make-dir');
+const bundleBuilder = require('./build-bundle.js');
+
+const LHROOT = path.join(__dirname, '..');
+const distDir = path.join(__dirname, '..', 'dist', 'lightrider');
+const sourceDir = __dirname + '/../clients/lightrider';
+
+const bundleOutFile = `${distDir}/report-generator.js`;
+const generatorFilename = `./lighthouse-core/report/report-generator.js`;
+
+const entrySourceName = 'lightrider-entry.js';
+const entryDistName = 'lighthouse-lr-bundle.js';
+
+makeDir.sync(path.dirname(distDir));
+
+/**
+ * @return {Promise<void>}
+ */
+async function copyAssets() {
+  return cpy([
+    'clients/lightrider/lightrider-ui-features.js',
+    'lighthouse-core/lib/file-namer.js',
```

Comment:
do you also want to copy all the other renderer assets LR has copied out manually right now? Or wait on this because you don't want to rewrite the matching bash script on the other side :)

Reply:
I think this works for now, because that whole file portion is copied wholesale.  Simplifying all of this syncing should def be a bug to track.